beacon-chain/blockchain/kzg/kzg.go

```diff
@@ -78,22 +72,29 @@ func ComputeBlobKZGProof(blob *Blob, commitment Commitment) (Proof, error) {
 
 	proof, err := kzg4844.ComputeBlobProof(&kzgBlob, kzg4844.Commitment(commitment))
 	if err != nil {
-		return [48]byte{}, err
+		return Proof{}, err
 	}
-	return Proof(proof), nil
+	return Proof(proof[:]), nil
```

Comment:
I'm not a big fan of casting a slice into an array this way, because it can panic: https://go.dev/play/p/4JCsoOR1xKu
The copy pattern does not panic: https://go.dev/play/p/A4_nO0p1jbG